.github/workflows/build-node-binaries.yaml

```diff
@@ -24,8 +24,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          # ref: ${{ github.event.inputs.git_ref }}
-          ref: aptos-node-v1.4.3
```

Comment:
oh my...how did this get in there?

Reply:
Testing 🤦 

We stopped using this because it made the full release flow too complicated